docs/overview/application.md

```diff
@@ -49,6 +49,8 @@ ExpressoTS will prevent you to do that as there are no listeners to handle incom
 
 The Application class offers a way of creating and configuring the server, passing **[Expressjs middlewares](https://expressjs.com/en/guide/writing-middleware.html)** or other middlewares upon server creation.
 
+It also provides a listen method that starts the server and waits for incoming requests. In the listen method, developers can define not only the port number, but also the server environment, which can be development, test, or production. Additionally, developers can set the application name and version to be displayed in the console when the server starts, as shown in the following example:
```

Comment:
This can be removed as this is described in the specific section from Listen Method

Reply:
The example will be removed too?